TSE REVIEW

carlos-ssh opened the pull request:
Regarding the latest review, we created a brand new repository and uploaded the code. Then we created a brand new branch to work on.

We also updated the link of the live demo in the readme file.
That will properly work once the branch is merged to the master.


We added the Linters configuration.

carlos-ssh committed:
added the code to a brand new repository and created a new branch to properly function on github

carlos-ssh committed:
the link will work properly once feature branch is merged to the master branch

carlos-ssh committed:
added the linters configuration

trekab reviewed (changes requested):
# Changes Required!
Hello @carlos-ssh and @wrakc.  Great job so far. However, you have some changes to make to this project.
## Mandatory changes:
- [x] Sign in button should be in the top right corner. Yours appears to have unnecessary white space on top.
- [x]  Fix your [HTML validator](https://validator.w3.org/#validate_by_upload) errors. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from pathwright.)
- [x] The form input for phone number should be of type "tel"(not phone).
- [ ] The Live demo link in your README file should link to your latest version of the project.
- [x] The intuit product row should have logos and text (logo on the left, text on the right) for Mint, Quickbooks and Turbotax. Your clone only has images.

## Optional:
- [ ] Your font-size is too small compared to the [original](https://accounts.intuit.com/signup.html).
- [ ] The 'reCAPTCHA' part on the form should be at the bottom-most part of the form.
- [ ] Make more use of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). The HTML validator will flag most of the semantic errors but a review of how to use those elements will improve your knowledge.

__Kindly work on the changes and submit another code review request__
_Feel free to contact me on slack for inquiries about this review_ :wink: 
__Treasure Kabareebe__

carlos-ssh committed:
we moved the signin button at the top-right of the page and we added text beside the logo of the intuit product

carlosveigadev committed:
fixed some linter errors

carlos-ssh committed:
fixed the recaptcha container

carlos-ssh commented:
- we fixed the sign in button in the top right corner.
- we fix errors in HTML with the HTML validator.
- we fix the form input for phone number using the parameter "tel".
- we fixed the product row using logos and text.

"THE  LIVE DEMO WILL BE WORKING WHEN WE MERGE TO THE MASTER BRANCH"
